packages/flutter/lib/src/services/sensitive_content.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/widgets.dart';
+library;
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// The level of sensitivity that content in the widget tree has.
```

Comment:
This reads awkwardly maybe 
```suggestion
/// The possible values for a widget tree's contentSensitivity. 
```